contrib/uptrace/bun/example_test.go

```diff
@@ -0,0 +1,31 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024 Datadog, Inc.
+
+package bun_test
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/uptrace/bun"
+	"github.com/uptrace/bun/dialect/sqlitedialect"
+	buntrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/uptrace/bun"
+	_ "modernc.org/sqlite"
```

Comment:
should we test using postgres to avoid this extra dependency?

Reply:
@rarguelloF `modernc.org/sqlite` was an indirect dependency already present in `go.mod`.

I'd rather to keep testing against SQLite and add tests against Postgres, to make sure everything works with the different supported databases (or a few of them).